cl/alerts/templates/alert_email.html

```diff
@@ -47,7 +47,8 @@ <h3 class="alt bottom" style="font-size: 1.5em; font-weight: normal; line-height
                         From the court
                     </a>
                     &nbsp;&nbsp;|&nbsp;&nbsp;
-                    <a href="https://www.courtlistener.com/{{result.local_path}}" style="font-size: 100%; font-weight: inherit; font-family: inherit; color: #009; border: 0; font-style: inherit; padding: 0; text-decoration: none; vertical-align: baseline; margin: 0;">
+                    {# Provide link to S3 for OA for now, for everything later. #}
+                    <a href="https://{% if type == "oa" %}storage{% else %}www{% endif %}.courtlistener.com/{{result.local_path}}" style="font-size: 100%; font-weight: inherit; font-family: inherit; color: #009; border: 0; font-style: inherit; padding: 0; text-decoration: none; vertical-align: baseline; margin: 0;">
```

Comment:
The compactness took me a minute to suss out.  I suppose the lack of spacing is important to URL generation but if it isn't I think there is a better way to write this.

Reply:
This is only temporary, for two weeks, until we swap opinions alerts over to S3 too, so I think it's OK.